go/teams/sig.go

```diff
@@ -57,6 +58,15 @@ func RootTeamIDFromNameString(name string) keybase1.TeamID {
 	return keybase1.TeamID(hex.EncodeToString(idBytes))
 }
 
+func NewImplicitTeamName() (string, error) {
+	dat, err := libkb.RandBytes(16)
```

Comment:
can you grab this from the constants file too?  It's too bad we can't do this in `extras.go` since it would seem similar to what's already there. But for now just use the `16` from that file.